controllers/util/secrets_util.go

```diff
@@ -230,7 +239,41 @@ func newS3ConfigurationSecret(s3SecretRef corev1.SecretReference, targetns strin
 	}
 }
 
-func newConfigurationPolicy(name string, object runtime.RawExtension) *cpcv1.ConfigurationPolicy {
+func newVeleroSecret(s3SecretRef corev1.SecretReference, velero VeleroConfiguration) *localSecret {
+	return &localSecret{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Secret",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "v" + s3SecretRef.Name,
+			Namespace: velero.Namespace,
+		},
+		Data: map[string]string{
+			velero.KeyName: "[default]\n" +
```

Comment:
This should change to:
```
'{{ (printf "default\n  aws_access_key_id = %s\n  aws_secret_access_key = %s"
  ((lookup "v1" "Secret" "ramen-system" "ramen-s3-secret-dr1").data.AWS_ACCESS_KEY_ID | base64dec)
  ((lookup "v1" "Secret" "ramen-system" "ramen-s3-secret-dr1").data.AWS_SECRET_ACCESS_KEY | base64dec)
) | base64enc }}'
```

The current method of processing the data does not work. The method above would copy a secret from the managed cluster, from one namespace to another, with the required format changes for velero. (So `ramen-system` as the namespace would actually be dr-cluster namespace on the managed cluster)

Reply:
Further to this, had to split it into 2 policies, as 2 secrets where one depends on the other being present, in the same policy does not work.